test/browser/pages/DVADetailsEntryPage.js

```diff
@@ -391,4 +402,36 @@ exports.DVADetailsEntryPage = class PlaywrightDevPage {
       validLicenceExample
     );
   }
+
+  async assertDVAConsentSentenceOne(consentFirstSentence) {
+    await this.page.waitForLoadState("domcontentloaded");
+    expect(await this.isCurrentPage()).to.be.true;
+    expect(await this.consentSectionSentenceOneDVA.innerText()).to.contains(
+      consentFirstSentence
+    );
+  }
+
+  async assertDVAConsentSentenceTwo(consentSecondSentence) {
+    await this.page.waitForLoadState("domcontentloaded");
+    expect(await this.isCurrentPage()).to.be.true;
+    expect(await this.consentSectionSentenceTwoDVA.innerText()).to.contains(
+      consentSecondSentence
+    );
+  }
+
+  async assertDVAConsentPrivacyLink(consentPrivacyLink) {
+    await this.page.waitForLoadState("domcontentloaded");
+    expect(await this.isCurrentPage()).to.be.true;
+    expect(await this.privacyPolicyDVALink.innerText()).to.equal(
+      consentPrivacyLink
+    );
+  }
+
+  async assertDVARetryErrorMessage(retryMessageHeading) {
+    await this.page.waitForLoadState("domcontentloaded");
+    expect(await this.isCurrentPage()).to.be.true;
+    expect(
+      await this.drivingLicenceDVARetryMessageHeading.innerText()
```

Comment:
Has this been declared drivingLicenceDVARetryMessageHeading?

Reply:
removed this method which is not used.